server/src/schemas/user.ts

```diff
@@ -3,6 +3,7 @@ import { z } from 'zod';
 const userSchema = z.object({
 	email: z.string().email(),
 	password: z.string().min(6),
+	displayName: z.string().min(2).max(50).optional(),
```

Comment:
Don't we have more items in this object? Like theme, languages ?

Reply:
Since the Supabase already has one user table for authentication I am using its table and I created one Settings table with the extra settings linked to one signed-up user.

I'll add one image of the tables to the PR description to display it better.

The other comments are related to the Settings page. I'll improve that page yet, I could remove the Settings page from the PR, but if you don't mind I'd like to keep that just to have the initial settings page.

Reply:
Ok